src/diffusers/pipelines/stable_diffusion_xl/pipeline_flax_stable_diffusion_xl.py

```diff
@@ -215,14 +215,15 @@ def _generate(
         else:
             if latents.shape != latents_shape:
                 raise ValueError(f"Unexpected latents shape, got {latents.shape}, expected {latents_shape}")
-        # scale the initial noise by the standard deviation required by the scheduler
-        latents = latents * params["scheduler"].init_noise_sigma
 
         # Prepare scheduler state
         scheduler_state = self.scheduler.set_timesteps(
             params["scheduler"], num_inference_steps=num_inference_steps, shape=latents.shape
         )
 
+        # scale the initial noise by the standard deviation required by the scheduler
+        latents = latents * scheduler_state.init_noise_sigma
```

Comment:
We need to set the init_noise_sigma atfter scaling

Reply:
Oh yes, great catch!